javascript-objects/main.js

```diff
@@ -0,0 +1,33 @@
+var student = {
+  firstName: 'Dre Sean',
+  lastName: 'Sutton',
+  age: 26
+};
+
+var fullName = student.firstName + ' ' + student.lastName;
+console.log('Full Name:', fullName);
+
+student.livesInIrvine = false;
+student.previousOccupation = 'Special Ed Bus Driver';
+console.log('my life:', student);
+
+var myCar = {
+  make: 'Honda',
+  model: 'Civic',
+  year: 2009
+};
+
+myCar.color = 'Blue';
+myCar.isConvertible = false;
```

Comment:
Please use square bracket notation for these lines, as requested in the instructions